### PR TITLE
Chore: cardano- db-sync@alonzo-white-1.4 & cardano-node@alonzo-purple-1.0 

### PR DESCRIPTION
**Pull request by jvieiro**
# Description

Updates cardano node and db-sync dependencies to support alonzo. Also adds db snapshots with new schema.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ARG UBUNTU_VERSION=20.04
 FROM ubuntu:${UBUNTU_VERSION} as haskell-builder
 ARG CABAL_VERSION=3.2.0.0
-ARG CARDANO_NODE_VERSION=1.27.0
-ARG CARDANO_DB_SYNC_VERSION=10.0.1
+ARG CARDANO_NODE_VERSION=alonzo-white-1.4
```
Comment (rhyslbw) [inline Dockerfile:4]:
```suggestion
ARG CARDANO_NODE_VERSION=alonzo-purple-1.0
```
The target tag is now `alonzo-purple-1.0`. Config for _alonzo-qa_ network is in https://github.com/input-output-hk/cardano-configurations/commit/ad34c4ef391124d4582e1bf22307950fa299f26b

File: Dockerfile [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
+ARG CARDANO_DB_SYNC_VERSION=alonzo-white-1.4
```
Comment (rhyslbw) [inline Dockerfile:5]: :+1:

File: cardano-rosetta-server/docker-compose.yml [hunk 3]
```diff
@@ -22,7 +22,7 @@ services:
         max-size: "200k"
         max-file: "10"
   cardano-node:
-    image: inputoutput/cardano-node:${CARDANO_NODE_VERSION:-1.27.0}
+    image: inputoutput/cardano-node:${CARDANO_NODE_VERSION:-alonzo-white-1.4}
```
Comment (rhyslbw) [inline cardano-rosetta-server/docker-compose.yml:25]:
```suggestion
    image: inputoutput/cardano-node:${CARDANO_NODE_VERSION:-alonzo-purple-1.0}
```

**Review by rhyslbw — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (jvieiro) [reply Dockerfile:4]: Is necessary to recreate the snapshots against that tag too?

Comment (rhyslbw) [reply Dockerfile:4]: It should be fine for this preliminary version, as the `cardano-db-sync` version has not changed.

File: Dockerfile [hunk 4]
```diff
@@ -1,8 +1,8 @@
 ARG UBUNTU_VERSION=20.04
 FROM ubuntu:${UBUNTU_VERSION} as haskell-builder
 ARG CABAL_VERSION=3.2.0.0
-ARG CARDANO_NODE_VERSION=1.27.0
-ARG CARDANO_DB_SYNC_VERSION=10.0.1
+ARG CARDANO_NODE_VERSION=alonzo-purple-1.0
+ARG CARDANO_DB_SYNC_VERSION=alonzo-white-1.4
```
Comment (rhyslbw) [inline Dockerfile:5]:
```suggestion
ARG CARDANO_DB_SYNC_VERSION=alonzo-purple-1.0
```

Comment (rhyslbw) [reply Dockerfile:5]: This tag was recently created to align with the target network. It's pointing to the same commit

File: cardano-rosetta-server/docker-compose.yml [hunk 5]
```diff
@@ -34,7 +34,7 @@ services:
         max-size: "400k"
         max-file: "20"
   cardano-db-sync:
-    image: inputoutput/cardano-db-sync:${CARDANO_DB_SYNC_VERSION:-10.0.1}
+    image: inputoutput/cardano-db-sync:${CARDANO_DB_SYNC_VERSION:-alonzo-white-1.4}
```
Comment (rhyslbw) [inline cardano-rosetta-server/docker-compose.yml:37]:
```suggestion
    image: inputoutput/cardano-db-sync:${CARDANO_DB_SYNC_VERSION:-alonzo-purple-1.0}
```

**Review by rhyslbw — CHANGES_REQUESTED** [review]

File: Dockerfile [hunk 6]
```diff
@@ -1,8 +1,8 @@
 ARG UBUNTU_VERSION=20.04
 FROM ubuntu:${UBUNTU_VERSION} as haskell-builder
 ARG CABAL_VERSION=3.2.0.0
-ARG CARDANO_NODE_VERSION=1.27.0
-ARG CARDANO_DB_SYNC_VERSION=10.0.1
+ARG CARDANO_NODE_VERSION=alonzo-purple-1.0
```
Comment (rhyslbw) [inline Dockerfile:4]:
```suggestion
ARG CARDANO_NODE_VERSION=alonzo-purple-1.0.1
```

File: cardano-rosetta-server/docker-compose.yml [hunk 7]
```diff
@@ -22,7 +22,7 @@ services:
         max-size: "200k"
         max-file: "10"
   cardano-node:
-    image: inputoutput/cardano-node:${CARDANO_NODE_VERSION:-1.27.0}
+    image: inputoutput/cardano-node:${CARDANO_NODE_VERSION:-alonzo-purple-1.0}
```
Comment (rhyslbw) [inline cardano-rosetta-server/docker-compose.yml:25]:
```suggestion
    image: inputoutput/cardano-node:${CARDANO_NODE_VERSION:-alonzo-purple-1.0.1}
```

**Pull request merged**